Fix colcon test-result command on CI

**Describe the bug**

```bash
Starting >>> pendulum_demo
--- stderr: pendulum_demo
Errors while running CTest
---
Finished <<< pendulum_demo [10.8s]	[ with test failures ]
Starting >>> pendulum_bringup
Finished <<< pendulum_bringup [1.65s]
Starting >>> pendulum
Finished <<< pendulum [0.03s]

Summary: 7 packages finished [26.3s]
  1 package had stderr output: pendulum_demo
  1 package had test failures: pendulum_demo
usage: colcon test-result [-h] [--test-result-base TEST_RESULT_BASE] [--all]
                          [--result-files-only] [--verbose] [--delete]
                          [--delete-yes] [--mixin-files [FILE [FILE ...]]]
                          [--mixin [mixin1 [mixin2 ...]]]
colcon test-result: error: argument --test-result-base: Path 'build' does not exist
The command '/bin/sh -c /tmp/pendulum/github-actions.sh' returned a non-zero code: 2

Error: Process completed with exit code 2.
```

--- a/github-actions.sh
+++ b/github-actions.sh
@@ -66,7 +66,7 @@ function compile_ws()
   fi
   echo -e "${YELLOW}Testing WS${RESET}"
   if ! colcon test --build-base build-install --install-base install-merge --merge-install --return-code-on-test-failure; then
-    colcon test-result --verbose
+    colcon test-result --test-result-base build-install --verbose
     echo -e "${RED}Error testing the ws${RESET}"
     exit 1
   else

